app/main.py

```diff
@@ -156,7 +156,7 @@ async def ocr_request_endpoint(request: OcrRequest):
     print(f"Processing PDF with strategy: {request.strategy}, ocr_cache: {request.ocr_cache}, model: {request.model}, storage_profile: {request.storage_profile}, storage_filename: {request.storage_filename}")
 
     # Asynchronous processing using Celery
-    task = ocr_task.apply_async(args=[file_content, request.strategy, "uploaded_file.pdf", pdf_hash, request.ocr_cache, request.prompt, request.model, request.storage_profile, request.storage_filename])
+    task = ocr_task.apply_async(args=[file_content, request.strategy, "uploaded_file.png", pdf_hash, request.ocr_cache, request.prompt, request.model, request.storage_profile, request.storage_filename])
```

Comment:
why this change?

Reply:
Only tests before I did universal default file (currently comming from file format).